src/Raven.Server/Documents/ShardedHandlers/ShardedCommands/FetchDocumentsFromShardsCommand.cs

```diff
@@ -5,20 +5,12 @@
 
 namespace Raven.Server.Documents.ShardedHandlers.ShardedCommands
 {
-    public class FetchDocumentsFromShardsCommand : ShardedCommand, IDisposable
+    public class FetchDocumentsFromShardsCommand : ShardedCommand
     {
-        private readonly IDisposable _disposable;
-        public TransactionOperationContext Context;
```

Comment:
why was it moved to `ShardedBaseCommand<T>` ?

Reply:
Ping cc @karmeli87

Reply:
removed